translated files/UI/ui_item.csv

```diff
@@ -1830,7 +1830,7 @@ WithdrawResultMessage2#0,"""アイテムを取り下げできませんでした
 Title#18,"""広告編集"""
 EditCmText#0,"""広告文を記入"""
 Decide#0,"""決定"""
-CancelText#0,"""キャンセル"""
+CCancelText#0,"""キャンセル"""
```

Comment:
I don't know. These files were handed to me by our translation team.
Now I am correcting all the mistakes that they made.
Later I will explain to them what they were wrong.